framework/src/modules/keys/keys_module.ts

```diff
@@ -143,23 +143,23 @@ export class KeysModule extends BaseModule {
 		genesisBlock,
 	}: AfterGenesisBlockApplyContext<AccountKeys>): Promise<void> {
 		const errors = [];
-		for (const account of genesisBlock.header.asset.accounts) {
+		for (const [index, account] of genesisBlock.header.asset.accounts.entries()) {
```

Comment:
Maybe related to other issue, but since this loop executes a lot, probably better to use
```
for (let index = 0; index < genesisBlock.header.asset.accounts.length; i += 1) {
  const account = genesisBlock.header.asset.accounts[i];
```